cfgmgr/intfmgr.cpp

```diff
@@ -660,3 +667,51 @@ void IntfMgr::doTask(Consumer &consumer)
         it = consumer.m_toSync.erase(it);
     }
 }
+
+void IntfMgr::doCfgVoqInbandInterfaceTask(Consumer &consumer)
```

Comment:
Why is this a separate function? looks like its just duplicated?

Reply:
Agreed. We'll remove this function and inband config handling will be moved to existing doTask().